src/platform_impl/windows/event_loop.rs

```diff
@@ -265,10 +265,34 @@ impl<T> EventLoopWindowTarget<T> {
     }
 }
 
+/// Returns the id of the main thread.
+///
+/// Windows has no real API to check if the current executing thread is the "main thread", unlike macOS.
+///
+/// Windows will let us look up the current thread's id, but there's no API that lets us check what the
+/// id of the main thread is. We would somehow need to get the main thread's id before a developer
+/// could spin off any other threads inside of the main entrypoint in order to emulate the capabilities
+/// of other platforms.
+///
+/// We can get the id of the main thread by using CRT initialization. CRT initialization can be used
+/// to setup global state within a program. The OS will call a list of function pointers which assign
+/// values to a static fields. To have get a hold of the main thread id, we need to place our
```

Comment:
```suggestion
/// values to a static variable. To have get a hold of the main thread id, we need to place our
```
not: The ID of the main thread is being assigned to a static variable, not a field of a struct.